onnx/defs/schema.cc

```diff
@@ -82,6 +85,12 @@ void OpSchema::CheckInputOutputType(struct InferenceContext& ctx) const {
         !all_types.empty() &&
         all_types.find(Utils::DataTypeUtils::ToType(*param_type)) ==
             all_types.end()) {
+      cout << "catch!" << endl;                         
```

Comment:
Is this meant to be for debugging?

Reply:
Yes. Sorry that it will be removed. The current version is still in progress.